code/bmpman/bmpman.h

```diff
@@ -45,7 +45,9 @@
  *   If anything we could/should reduce MAX_BITMAPS in the future.  Where it's at now should accomidate even the
  *   largest mods.  --  Taylor
  */
-#define MAX_BITMAPS 4750
+#define DEFAULT_MAX_BITMAPS 4750
+
+extern int MAX_BITMAPS;
```

Comment:
Since this is basically the size of an array it should be a `size_t`.

Reply:
For anyone who didn't follow the discussion on Discord, changing it to `size_t` causes multiple problems (like crashing in certain functions if the handle is negative), so it'll be staying an `int`.